packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -184,15 +184,20 @@ export default async function (plop: NodePlopAPI) {
           return new Promise((resolve, reject) => {
             const {project} = data as PlopData;
             const process = spawn(getPackageManager(), ['install'], {
-              stdio: 'ignore',
+              stdio: ['ignore', 'ignore', 'pipe'],
               cwd: join(currentPath, project),
             });
 
+            let error = '';
+            process.stderr.on('data', (chunk) => {
+              error += chunk.toString();
+            });
```

Comment:
Logging just the stderr won't be enough IMO.
You should log stdout too for full context (Yeah, it'll be noisy, but when 💩 hit the fan, I think it's best to have too much than not enough info).